### ISSUE TITLE
Inconsistent behavior for `original-fs` of the rule `prefer-node-protocol`

### ISSUE THREAD

**Issue by OpportunityLiu** [sindresorhus/eslint-plugin-unicorn#2143]
Since https://github.com/sindresorhus/builtin-modules uses `const {builtinModules} = require('module');` to load a dynamic list of built in modules, the behavior differs from running `eslint` in cli and reported from vs code plugin.

```js
import { createWriteStream } from 'original-fs';
```

I'm not sure if it is a bug of this repo, or should I report it to https://github.com/Microsoft/vscode-eslint?

**Comment by subframe7536** [comment]
same here

**Comment by fregante** [comment]
> 'm not sure if it is a bug of this repo

The question that should be answered is:

- what leads `module` to be different?

Is the editor loading the wrong version? Is the editor using the locally-installed version of eslint and unicorn or a global version?

My guess is that it's not using what's in your node_modules, so that's a configuration/editor issue rather than a unicorn issue.

**Comment by fregante** [comment]
A solution to this would be to _optionally_ load the [static list](https://github.com/sindresorhus/builtin-modules?tab=readme-ov-file#static-list) instead of the one provided by `module`, but I'll let Unicorn's maintainer determine if that's worth it.
